packages/syft/src/syft/service/code/user_code.py

```diff
@@ -395,13 +396,16 @@ def debox_asset(arg: Any) -> Any:
 
 def syft_function(
     input_policy: Union[InputPolicy, UID],
-    output_policy: Union[OutputPolicy, UID],
+    output_policy: Union[OutputPolicy, UID] = None,
```

Comment:
Q: Maybe we could enforce the users to always provide a output policy?,
As it would enforce them to also understand the restrictions that the code would be processed.
It would also make the users be context aware while choosing policies.